src/axolotl/prompt_strategies/dpo/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""
+module for DPO style dataset transform strategies
+"""
+
+import importlib
+
+
+def load(strategy, cfg):
+    try:
+        load_fn = strategy.split(".")[-1]
```

Comment:
This is most likely not correct. The `strategy` includes underscores, not `.`, such as `intel_apply_chatml`.

Reply:
```python
def load(strategy, cfg):
    try:
        load_fn = strategy.split("_")[-1]
        #strategy = ".".join(strategy.split("_")[:-1])
        LOG.info(load_fn)
        LOG.info(strategy)
        mod = importlib.import_module(f".{load_fn}", "axolotl.prompt_strategies.dpo")
        func = getattr(mod, strategy)
        load_kwargs = {}
        return func(cfg, **load_kwargs)
    except Exception as e:  # pylint: disable=broad-exception-caught
        LOG.info(e)
        return None
```

Reply:
This works for me.

Reply:
the intention is the setting is something like
```yaml
type: chatml.argilla
```

in which case it will load the argilla function from the `axolotl.prompt_strategies.dpo.chatml` module.

Reply:
Hi @winglian 👋🏻 thanks. That makes sense. I will test it later today 👍🏻